src/python/visclaw/geoplot.py

```diff
@@ -196,15 +211,25 @@ def surface_or_depth(current_data):
     h = q[0,:,:]
     eta = q[3,:,:]
     topo = eta - h
-    surface = ma.masked_where(h<=drytol, eta)
-    depth = ma.masked_where(h<=drytol, h)
-    surface_or_depth = where(topo<0, surface, depth)
+
+    # With this version, the land was plotted as white in png files for KML.
+    # surface = ma.masked_where(h <= drytol, eta)
+    # depth = ma.masked_where(h <= drytol, h)
+    # surface_or_depth = where(topo<0, surface, depth)
+
+    # With this version, the land is transparent.
```

Comment:
That was the point - the masking before didn't work for GE.    Is this new masking a problem for standard output?